docs/existing-aws-infrastructure.md

```diff
@@ -0,0 +1,94 @@
+# Consuming Existing AWS Infrastructure
+
+Normally, Cluster API will create infrastructure on AWS when standing up a new workload cluster. However, it is possible to have Cluster API re-use existing AWS infrastructure instead of creating its own infrastructure. Follow the instructions below to configure Cluster API to consume existing AWS infrastructure.
+
+## Prerequisites
+
+In order to have Cluster API consume existing AWS infrastructure, you will need to have already created the following resources:
+
+* A VPC
+* One or more private subnets (subnets that do not automatically assign a public IP address to EC2 instances)
```

Comment:
What CAPA considers public is mainly whether the subnet in question has a route to an internet gateway – if yes, it's public, otherwise it's private.

Reply:
Added some text to that effect via 38ce860